schema/create_raw_employee_data.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS raw_employee_data (
+    employee_id VARCHAR(500),
+    first_name VARCHAR(500),
+    last_name VARCHAR(500),
+    department_id VARCHAR(500),
+    department_name VARCHAR(500),
+    manager_employee_id VARCHAR(500),
+    employee_role VARCHAR(500),
+    salary VARCHAR(500),
+    hire_date VARCHAR(500),
+    terminated_date VARCHAR(500),
+    terminated_reason VARCHAR(500),
+    dob VARCHAR(500),
+    fte VARCHAR(500),
+    location VARCHAR(500)
+);
```

Comment:
Why do we have the same table schema?

Reply:
Didi, I created a branch from hijo ko extraction, so these are the files from yesterday's task.